thumbor/thumbor.conf

```diff
@@ -1,260 +1,221 @@
+## faces. The default should be fine, so change this at your own peril. if you
+## set a relative path it will be relative to the
+## thumbor/detectors/face_detector folder
+# FACE_DETECTOR_CASCADE_FILE = "haarcascade_frontalface_alt.xml"
+
+## This is the security key used to encrypt/decrypt urls.  make sure this is
+## unique and not well-known This can be any string of up to 16 characters
 SECURITY_KEY = "MY_SECURE_KEY"
 
-# if you enable this, the unencryted URL will be available
-# to users.
-# IT IS VERY ADVISED TO SET THIS TO False TO STOP OVERLOADING
-# OF THE SERVER FROM MALICIOUS USERS
+## If you enable this, the unencryted URL will be available to users. IT IS
+## VERY ADVISED TO SET THIS TO False TO STOP OVERLOADING OF THE SERVER FROM
+## MALICIOUS USERS
 ALLOW_UNSAFE_URL = True
 
-# Mixed storage classes. Change them to the fullname of the
-# storage you desire for each operation.
-#MIXED_STORAGE_FILE_STORAGE = 'thumbor.storages.file_storage'
-#MIXED_STORAGE_CRYPTO_STORAGE = 'thumbor.storages.no_storage'
-#MIXED_STORAGE_DETECTOR_STORAGE = 'thumbor.storages.no_storage'
-
-
-# This configuration indicates whether thumbor should use a
-# custom error handler. 
-# Defaults to: False
-#USE_CUSTOM_ERROR_HANDLING=False
-#ERROR_HANDLER_MODULE='thumbor.error_handlers.sentry'
-#SENTRY_DSN_URL=''
-#SENTRY_ENVIRONMENT=None
-
-#FILTERS = [
-    #'thumbor.filters.brightness',
-    #'thumbor.filters.colorize',
-    #'thumbor.filters.contrast',
-    #'thumbor.filters.rgb',
-    #'thumbor.filters.round_corner',
-    #'thumbor.filters.quality',
-    #'thumbor.filters.noise',
-    #'thumbor.filters.watermark',
-    #'thumbor.filters.equalize',
-    #'thumbor.filters.fill',
-    #'thumbor.filters.saturation',
-    #'thumbor.filters.sharpen',
-    #'thumbor.filters.strip_exif',
-    #'thumbor.filters.strip_icc',
-    #'thumbor.filters.frame',
-    #'thumbor.filters.grayscale',
-    #'thumbor.filters.rotate',
-    #'thumbor.filters.format',
-    #'thumbor.filters.max_bytes',
-    #'thumbor.filters.no_upscale',
-
-    ## can only be applied if there are already points for the image being served
-    ## this means that either you are using the local face detector or the image
-    ## has already went through remote detection
-    ## 'thumbor.filters.redeye',
-#]
-
-#OPTIMIZERS = [
-    #'thumbor.optimizers.jpegtran',
-    #'thumbor.optimizers.gifv',
-#]
+## Mixed storage classes. Change them to the fullname of the storage you desire
+## for each operation.
+# MIXED_STORAGE_FILE_STORAGE = "thumbor.storages.file_storage"
+# MIXED_STORAGE_CRYPTO_STORAGE = "thumbor.storages.no_storage"
+# MIXED_STORAGE_DETECTOR_STORAGE = "thumbor.storages.no_storage"
+
+
+## This configuration indicates whether thumbor should use a custom error
+## handler. Defaults to: False
+# USE_CUSTOM_ERROR_HANDLING=False
+# ERROR_HANDLER_MODULE="thumbor.error_handlers.sentry"
+# SENTRY_DSN_URL=""
+# SENTRY_ENVIRONMENT=None
+
+FILTERS = BUILTIN_FILTERS
```

Comment:
Not sure if `FILTERS` should be set explicitly. Currently it automatically would load "all by default" anyway.

Does the `thumbor-doctor` depend on this, too?

@heynemann what do you think?

Reply:
```suggestion
# FILTERS = BUILTIN_FILTERS
```

I like that now there's just the BUILTIN_FILTERS, but it could be commented out as you suggest.